### ISSUE TITLE
Add Icloud contact cards to contacts automatically 

### ISSUE THREAD

**Issue by cameronaaron** [BlueBubblesApp/bluebubbles-server#646]
You developed an API that verifies whether a user is sharing a contact card and another that retrieves it. Additionally, you have an API that allows users to share their own contact cards. it would be beneficial if these contact cards could be exported directly to the phone's contacts book.

**Comment by zlshames** [comment]
This sounds like it's an app feature, not a server feature. I'm going to move it to the server, unless you have any objections. Let me know so I can move it over.

**Comment by cameronaaron** [comment]
Yeah maybe in this case it would be better to do it app side. I do think a lot of the other contact related functionality should be handled on server like since we sign in to server already it makes more sense to just have the contacts directly get pulled from google instead of the client having to send them. This would also improve things like beeper and other apps that use the api

**Comment by zlshames** [comment]
> Yeah maybe in this case it would be better to do it app side. I do think a lot of the other contact related functionality should be handled on server like since we sign in to server already it makes more sense to just have the contacts directly get pulled from google instead of the client having to send them. This would also improve things like beeper and other apps that use the api

I get that having the server pull from Google Contacts _makes sense_, however, this request is asking for something different
